pkg/log/options.go

```diff
@@ -24,15 +24,50 @@ import (
 const (
 	// None is used to disable logging output as well as to disable stack tracing.
 	None zapcore.Level = 100
+
+	defaultOutputLevel        = zapcore.InfoLevel
+	defaultStackTraceLevel    = None
+	defaultOutputPath         = "stdout"
+	defaultErrorOutputPath    = "stderr"
+	defaultRotationMaxAge     = 30
+	defaultRotationMaxSize    = 100 * 1024 * 1024
+	defaultRotationMaxBackups = 1000
 )
 
 // Options defines the set of options supported by Istio's component logging package.
 type Options struct {
 	// OutputPaths is a list of file system paths to write the log data to.
 	// The special values stdout and stderr can be used to output to the
-	// standard I/O streams.
+	// standard I/O streams. THis defaults to stdout.
 	OutputPaths []string
 
+	// ErrorOutputPaths is a list of file system paths to write logger errors to.
+	// The special values stdout and stderr can be used to output to the
+	// standard I/O streams. THis defaults to stderr.
```

Comment:
`This` again

Reply:
Fixed, along with a few other comment errors I noticed.